docs/plugins/synoindex.rst

```diff
@@ -0,0 +1,10 @@
+synoindex Plugin
+================
+
+``synoindex`` is a very simple plugin for beets that notifies the Synology
+DiskStation indexing service to index files that are imported into the beets
+library.  This in turn enables new music to be immediately accessible in Audio
+Station.
+
+To use ``synoindex`` plugin, enable it in your configuration
+(see :ref:`using-plugins`).
```

Comment:
Maybe it would be worth noting here that this works by running the `synoindex` command, so the plugin assumes you're running _on the Synology_, where that command's available—hence the lack of configurable option. Maybe that's obvious, but clarity never hurts!